### ISSUE TITLE
[Container]: All content must reside in HTML5 sectioning element

### ISSUE THREAD

**Issue by jendowns** [carbon-design-system/gatsby-theme-carbon#563]
## Detailed description

The main content of any/every page is not wrapped a `<main>` or `<section role="main">` (i.e., an HTML5 sectioning element), therefore numerous DAP violations are generated for every heading or paragraph that appears on the page, outside of the required HTML5 sectioning element.

For example, there are related violations generated on the demo site's homepage:

![Screen Shot 2019-11-25 at 9 32 53 AM](https://user-images.githubusercontent.com/9057921/69554243-1d8c3f00-0f67-11ea-83df-be868efb4432.png)

A side effect of this is a separate violation for having a `<footer>` without a "main" HTML5 sectioning element, shown in this violation on every page:

![Screen Shot 2019-11-25 at 9 33 05 AM](https://user-images.githubusercontent.com/9057921/69554316-3a287700-0f67-11ea-96fc-fdb5af9e6dd6.png)

### What WCAG 2.1 checkpoint does the issue violate?

* [Checkpoint 2.4.1](http://www.ibm.com/able/guidelines/ci162/bypass_blocks.html)

### Proposed solution

Seems that a change will need to be made to the `Container` component. Right now it is just a plain `<div>`.

**Comment by jendowns** [comment]
I will propose a fix for this shortly 👍
